cli/cmd/logging.go

```diff
@@ -48,6 +48,6 @@ func initLogging(noLogFile bool, logLevel *enum.Enum, logFormat *enum.Enum, logC
 		}
 	}
 	mw := io.MultiWriter(writers...)
-	log.Logger = zerolog.New(mw).Level(zerologLevel).With().Str("module", "cli").Timestamp().Logger()
+	log.Logger = zerolog.New(mw).Level(zerologLevel).With().Str("module", "cli").Str("log-id", invocationUUID.String()).Timestamp().Logger()
```

Comment:
```suggestion
	log.Logger = zerolog.New(mw).Level(zerologLevel).With().Str("module", "cli").Str("trace-id", invocationUUID.String()).Timestamp().Logger()
```

Maybe a better name? Also maybe have a hidden flag/env variable to configure it incase one would like to have predictable trace IDs

Reply:
Unless I missed something, I don't think that this gets propagated to our traces. So maybe something like `invocation-id` would be better.